packages/react-core/src/components/AboutModal/examples/AboutModal.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: About modal
 section: components
-cssPrefix: pf-c-about-modal-box
+cssPrefix: pf-v5-c-about-modal-box
```

Comment:
Do we want to update this in example files right now? Looks like the CSS variables tables in component pages is empty in the preview. cc @nicolethoen would this be tied to https://github.com/patternfly/patternfly-org/issues/3489 ?

Reply:
Yeah I think that with https://github.com/patternfly/patternfly-org/issues/3489 the tables will work with this change.

Reply:
agreed! I think to keep parity with core, we should use the versioned prefixes here.